.editorconfig

```diff
@@ -413,5 +413,21 @@ dotnet_diagnostic.IDE0300.severity = suggestion
 dotnet_diagnostic.IDE0301.severity = suggestion
 dotnet_diagnostic.IDE0305.severity = suggestion
 
+
+# Remove unnecessary equality operator  IDE0100 (https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/style-rules/ide0100)
+# Making it a suggestion to have a flexibility for future
+dotnet_diagnostic.IDE0100.severity = suggestion
+
+# File header should match the template, making it error since couple of files met in the code base without any header
+dotnet_diagnostic.IDE0073.severity = error
+
 # Temporarily disable SA1010 "Opening square brackets should not be preceded by a space" until https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3687 is fixed
 dotnet_diagnostic.SA1010.severity = none
+
+
+#TODO:fix all xUnit new warning 
+dotnet_diagnostic.xUnit1031.severity = suggestion
+dotnet_diagnostic.xUnit1048.severity = suggestion
+dotnet_diagnostic.xUnit2020.severity = suggestion
+dotnet_diagnostic.xUnit1012.severity = suggestion
+dotnet_diagnostic.xUnit2029.severity = suggestion
```

Comment:
we disable one xunit analyzer in `eng/Common.globalconfig` already; move these there?

Reply:
This is removed completely, since the version of xunit was set explicitly. 
And will be upgraded once this PR is merged